rotkehlchen/tests/unit/test_cost_basis.py

```diff
@@ -1047,3 +1060,80 @@ def test_taxable_acquisition(accountant):
     assert pot.processed_events[0].free_amount == ZERO
     assert pot.processed_events[0].pnl.taxable == ONE
     assert pot.processed_events[0].pnl.free == ZERO
+
+
+@pytest.mark.parametrize('force_no_price_found_for', [[
+    (A_ETH, Timestamp(1677593073)),
+    (A_ETH, Timestamp(1677593074)),
+    (A_ETH, Timestamp(1677593075)),
+    (A_ETH, Timestamp(1677593076)),
+]])
+@pytest.mark.parametrize('db_settings,expected_pnls', [
+    [{'cost_basis_method': CostBasisMethod.FIFO}, [ZERO, ZERO, FVal(-10), FVal(3500), ZERO, FVal(-10), ZERO, ZERO, FVal(-10), FVal(1600), ZERO, FVal(-10)]],  # noqa: E501
+    [{'cost_basis_method': CostBasisMethod.ACB}, [ZERO, ZERO, FVal(3495), ZERO, ZERO, ZERO, FVal(-6), ZERO]],  # noqa: E501
+])
+def test_fees(accountant: 'Accountant', expected_pnls: list[FVal]):
+    """
+    Tests that fees are properly either calculated as standalone events or included in the price.
+    Values for the example are taken from https://github.com/rotki/rotki/issues/5561#issuecomment-1423338938.
+
+    Note: pnls are $10 higher than in the issue since examples in the issue say that transaction
+    costs should be subtracted when selling, but in fact transaction costs are always about
+    buy cost basis https://discord.com/channels/657906918408585217/683245846070165504/1077994503849386054.
```

Comment:
What? That sounds confusing.

1. Do not write things that are intended for the reviewer as part of the docstring. Comment on the PR itself. This won't stay in the docstring after the review.
2. What are you talking aout? Which example? That comment is huge and has different examples written by me based on two different examples from two different pdfs.

```suggestion
    Note: pnls are $10 higher than in the issue since examples in the issue say that transaction
    costs should be subtracted when selling, but in fact transaction costs are always about
    buy cost basis https://discord.com/channels/657906918408585217/683245846070165504/1077994503849386054.
```

Reply:
I meant Canada example